### DISCUSSION TITLE
Bitti mi

### DISCUSSION THREAD

**Discussion by Mehmetturgut1914** [IBAX-io/go-ibax#1828] [Soru Cevap - Türkçe]
İbax airdrop bitt mi Napıcaz şimdi ne olacak

**Comment by Yusufknk** [comment]
Evet bitti.  Dağıtım olacak (yılın ikinci çeyreğinde) sonra ön satış olacak. Bu süre zarfında bir kampanya & etkinlik daha olacak lütfen takipte kalın.

**Reply by Celoba223** [reply]
aga şubatta piyasaya sürülecek falan dendi ne oldu şimdi ben anlamadım

**Comment by Abdullahoda** [comment]
İlk airdrop bitti ikinci bir airdrop olucak mesajımı cevaplandı olarak işaretlersen sevinirim kardeşim
